[2025-05-16T08:52:55.830+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-16T08:52:55.839+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.get_required_files_and_download scheduled__2025-05-16T08:20:00+00:00 [queued]>
[2025-05-16T08:52:55.843+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.get_required_files_and_download scheduled__2025-05-16T08:20:00+00:00 [queued]>
[2025-05-16T08:52:55.843+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-16T08:52:55.847+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_required_files_and_download> on 2025-05-16 08:20:00+00:00
[2025-05-16T08:52:55.851+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=11725) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-16T08:52:55.852+0000] {standard_task_runner.py:72} INFO - Started process 11726 to run task
[2025-05-16T08:52:55.852+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DYNAMIC_FILE_TO_DB_LOADER', 'get_required_files_and_download', 'scheduled__2025-05-16T08:20:00+00:00', '--job-id', '1547', '--raw', '--subdir', 'DAGS_FOLDER/***_dynamic_file_to_db_loader.py', '--cfg-path', '/tmp/tmpdsmt4jpw']
[2025-05-16T08:52:55.853+0000] {standard_task_runner.py:105} INFO - Job 1547: Subtask get_required_files_and_download
[2025-05-16T08:52:55.874+0000] {task_command.py:467} INFO - Running <TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.get_required_files_and_download scheduled__2025-05-16T08:20:00+00:00 [running]> on host 65a84f9e3a41
[2025-05-16T08:52:55.908+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DYNAMIC_FILE_TO_DB_LOADER' AIRFLOW_CTX_TASK_ID='get_required_files_and_download' AIRFLOW_CTX_EXECUTION_DATE='2025-05-16T08:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-16T08:20:00+00:00'
[2025-05-16T08:52:55.909+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-16T08:52:55.918+0000] {base.py:84} INFO - Retrieving connection 'minioic'
[2025-05-16T08:52:55.918+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minioic', conn_type='aws') credentials retrieved from login and password.
[2025-05-16T08:52:56.066+0000] {s3.py:1467} INFO - Downloading source S3 file from Bucket ahajiyev with path abc.xlsx
[2025-05-16T08:52:56.123+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-05-16T08:52:56.126+0000] {s3.py:1467} INFO - Downloading source S3 file from Bucket ahajiyev with path xyz.xlsx
[2025-05-16T08:52:56.131+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-05-16T08:52:56.132+0000] {hook.py:208} WARNING - Data lineage tracking is disabled. Register a hook lineage reader to start tracking hook lineage.
[2025-05-16T08:52:56.133+0000] {airflow_dynamic_file_to_db_loader.py:162} INFO - Downloaded Mappings: [{'source_file_name': 'abc.xlsx', 'target_table_name': 'abc'}, {'source_file_name': 'xyz.xlsx', 'target_table_name': 'xyz'}]
[2025-05-16T08:52:56.133+0000] {python.py:240} INFO - Done. Returned value was: [{'source_file_name': 'abc.xlsx', 'target_table_name': 'abc'}, {'source_file_name': 'xyz.xlsx', 'target_table_name': 'xyz'}]
[2025-05-16T08:52:56.155+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-16T08:52:56.159+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=DYNAMIC_FILE_TO_DB_LOADER, task_id=get_required_files_and_download, run_id=scheduled__2025-05-16T08:20:00+00:00, execution_date=20250516T082000, start_date=20250516T085255, end_date=20250516T085256
[2025-05-16T08:52:56.197+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-16T08:52:56.229+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-05-16T08:52:56.230+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
